CHANGELOG.unreleased.md

```diff
@@ -41,8 +41,11 @@
 
 <!--packages-start-->
 
+- @xen-orchestra/xapi major
+- @xen-orchestra/mixins major
 - vhd-lib patch
 - @xen-orchestra/fs patch
+- @xen-orchestra/backups minor
```

Comment:
I believe something went wrong with the rebase.

I think you can simply remove these changes.